ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -121,4 +129,35 @@ boundary server -config="${this.configFilePath || '<path>'}/pki-worker.hcl"`;
       styleActiveLine: false,
     };
   }
+
+  //=methods
+
+  convertCommaSeparatedValuesToArray(input) {
+    return input
+      .split(',')
+      .filter((value) => value.trim())
+      .map((value) => `"${value?.trim()}"`)
+      .join(', ');
+  }
+
+  getTagConfigString() {
+    return this.workerTags
+      .map(
+        (tag) =>
+          `${tag.key} = [${this.convertCommaSeparatedValuesToArray(tag.value)}]`
+      )
+      .join('\n    ');
```

Comment:
would`\t` work for the spacing here? feel like that might read better

Reply:
I kept it all spaces since that's what the rest of the config is using. I tried not to mix tabs and spaces while building up the config